global_vars.tf

```diff
@@ -0,0 +1,21 @@
+variable "env" {
+  description = "The environment you wish to use"
+}
+
+variable "aws_secret_key" {
+  description = "Amazon Web Service Secret Key"
+}
+
+variable "aws_access_key" {
+  description = "Amazon Web Service Access Key"
+}
+
+variable "read_capacity" {
```

Comment:
I would add the table name to this variable. so it won't need changing if we add more tables.